documentation/user_docs/docs/manual/List_of_functions.rst

```diff
@@ -0,0 +1,127 @@
+- :ref:`disp2sqw_eval <manual/Simulation:Simulation>`
+- :ref:`dispersion_plot <manual/Simulation:Simulation>`
+- :ref:`disp2sqw_plot <manual/Simulation:Simulation>`
+
+.. _LoF_Fitting:
+
+:ref:`Fitting data <manual/Multifit:Multifit>`
+
+- :ref:`multifit <manual/Multifit:multifit>`
+- :ref:`multifit_func <manual/Multifit:multifit_func>`
+- :ref:`multifit_sqw <manual/Multifit:multifit_sqw>`
+- :ref:`multifit_sqw_sqw <manual/Multifit:multifit_sqw_sqw>`
+
+
+
+:ref:`Binary operations <manual/Binary_operations:Binary operations>`
+
+- :ref:`minus <manual/Binary_operations:List of operations and their equivalent code>`
+- :ref:`plus <manual/Binary_operations:List of operations and their equivalent code>`
+- :ref:`mtimes <manual/Binary_operations:List of operations and their equivalent code>`
+- :ref:`mrdivide <manual/Binary_operations:List of operations and their equivalent code>`
+- :ref:`mldivide <manual/Binary_operations:List of operations and their equivalent code>`
+- :ref:`mpower <manual/Binary_operations:List of operations and their equivalent code>`
+
+
+:ref:`Unary operations <manual/Unary_operations:Unary operations>`
+
+- :ref:`uplus <manual/Unary_operations:unary plus>`
+- :ref:`uminus <manual/Unary_operations:unary minus>`
+- :ref:`Trigonometric and hyperbolic functions <manual/Unary_operations:Trigonometric and hyperbolic functions>`
+- :ref:`Other mathematical functions <manual/Unary_operations:Other mathematical functions>`
+
+
+:ref:`Reshaping etc <manual/Reshaping_etc:Reshaping etc>`
+
+- :ref:`replicate <manual/Reshaping_etc:replicate>`
+- :ref:`compact <manual/Reshaping_etc:compact>`
+- :ref:`permute <manual/Reshaping_etc:permute>`
+- :ref:`cut <manual/Reshaping_etc:cut>`
+- :ref:`smooth <manual/Reshaping_etc:smooth>`
+- :ref:`mask <manual/Reshaping_etc:mask>`
+- :ref:`mask_points <manual/Reshaping_etc:mask_points>`
+- :ref:`mask_runs <manual/Reshaping_etc:mask_runs>`
+- :ref:`section <manual/Reshaping_etc:section>`
+
+
+:ref:`Read or write to disk <manual/Read_or_write_to_disk:Read or write to disk>`
+
+- :ref:`read <manual/Read_or_write_to_disk:read_horace, read_sqw, read_dnd>`
+- :ref:`save <manual/Read_or_write_to_disk:save>`
+- :ref:`save xye ascii file <manual/Read_or_write_to_disk:save_xye>`
+- :ref:`header <manual/Read_or_write_to_disk:head_horace, head_sqw, head_dnd>`
+
+..
+   - :ref:`display <manual/Read_or_write_to_disk:display>`
+
+
+:ref:`Changing object type <manual/Changing_object_type:Changing object type>`
+
+- :ref:`d0d <manual/Changing_object_type:d0d>`
+- :ref:`d1d <manual/Changing_object_type:d1d>`
+- :ref:`d2d <manual/Changing_object_type:d2d>`
+- :ref:`d3d <manual/Changing_object_type:d3d>`
+- :ref:`d4d <manual/Changing_object_type:d4d>`
+- :ref:`sqw <manual/Changing_object_type:sqw>`
+- :ref:`IX_dataset_1d <manual/Changing_object_type:IX_dataset_1d>`
+- :ref:`IX_dataset_2d <manual/Changing_object_type:IX_dataset_2d>`
+- :ref:`IX_dataset_3d <manual/Changing_object_type:IX_dataset_3d>`
```

Comment:
Text referred to here says an appropriately dimensioned sqw/dnd will be converted into the generic Herbert IX_dataset_nd. A brief description of what that is and why it is useful would be good

Reply:
To be honest, I'm not sure why users would need to know this... I think it's only used for some plots... (but it should be transparent to users).